### PR TITLE
twister: fix reporting and example in synopsis

### LINKED ISSUE #54454: Twister summary in some cases provides an irrelevant example
Twister provides a summary of failures and examples how to reproduce them, but the example given is not actually a failure. 

For example:
```
INFO    - The following issues were found (showing the top 10 items):
INFO    - 1) tests/lib/c_lib/libraries.picolibc on qemu_x86 error (Build failure)
INFO    - 2) tests/lib/cbprintf_fp/lib.cbprintf_fp.printf.picolibc on qemu_x86 error (Build failure)
INFO    - 3) tests/lib/cbprintf_fp/lib.cbprintf_fp.printfcb.picolibc on qemu_x86 error (Build failure)
INFO    - 4) tests/lib/mem_alloc/libraries.libc.picolibc.mem_alloc on qemu_x86 error (Build failure)
INFO    - 5) tests/lib/sprintf/libraries.picolibc.sprintf on qemu_x86 error (Build failure)
INFO    - 6) tests/lib/sprintf/libraries.picolibc.sprintf_new on qemu_x86 error (Build failure)
INFO    - 7) tests/misc/print_format/printk.format.picolibc on qemu_x86 error (Build failure)
INFO    - 
INFO    - To rerun the tests, call twister using the following commandline:
INFO    - ./scripts/twister -p <PLATFORM> -s <TEST PATH>, for example:
INFO    - 
INFO    - ./scripts/twister -p qemu_x86_64 -s tests/benchmarks/app_kernel/benchmark.kernel.application.fp.x86.sse
INFO    - or with west:
INFO    - west build -b qemu_x86_64 -T tests/benchmarks/app_kernel/benchmark.kernel.application.fp.x86.sse
INFO    - -+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
```

### PR DESCRIPTION
**Pull request by nashif**
Take an instance that has failed and not whatever was last processed in
the for loop.

Also, change instructions to use `west twister` rather than
`./scripts/twister` to be consistent.

Fixes #54454 

Signed-off-by: Anas Nashif <anas.nashif@intel.com>

### CODE REVIEW

**Comment by laurenmurphyx64** [comment]
@nashif Need a tiny bug report :bug: :lady_beetle:

File: scripts/pylib/twister/twisterlib/reports.py [hunk 1]
```diff
@@ -425,17 +425,18 @@ def synopsis(self):
...
+                example_instance = instance
             if cnt == 10:
                 break
 
-        if cnt:
+        if cnt and example_instance:
             logger.info("")
             logger.info("To rerun the tests, call twister using the following commandline:")
-            logger.info("./scripts/twister -p <PLATFORM> -s <TEST PATH>, for example:")
+            logger.info("west twister -p <PLATFORM> -s <TEST PATH>, for example:")
```
Comment (PerMac) [inline scripts/pylib/twister/twisterlib/reports.py:435]: I think it should be \<TEST ID\> instead of a \<TEST PATH\> as an argument for "-s"

**Review by PerMac — DISMISSED** [review]

**Review by hakehuang — DISMISSED** [review]

**Review by mbolivar-nordic — APPROVED** [review]

**Review by hakehuang — APPROVED** [review]

**Comment by mbolivar-nordic** [comment]
Still needs a bug report before we can merge

**Comment by nashif** [comment]
bug created.

**Pull request merged**